libopae/plugins/xfpga/metrics/metrics_utils.c

```diff
@@ -795,6 +795,32 @@ fpga_result get_fpga_object_type(fpga_handle handle,
 	return resval;
 }
 
+void *metrics_load_bmc_lib(void)
+{
+	char plugin_path[PATH_MAX];
+	const char *search_paths[] = {
+		"/usr/lib64/opae/",
```

Comment:
#define oape paths

Reply:
These appear only once within the given library.